domain/input.json

```diff
@@ -106,6 +106,13 @@
                 "key": "space",
                 "at": null
             }
+        },
+        {
+            "name": "pan-fry",
```

Comment:
Remove this from this PR since it is not part of this PR, so that we have less merge conflicts in the future